loader/loader.c

```diff
@@ -2631,6 +2630,8 @@ static VkResult loader_add_layer_properties(const struct loader_instance *inst,
             } while (layer_node != NULL);
         }
     }
+    
```

Comment:
This now leaks at line 2506, 2607. I see what you're saying with line 2587.  But you should probably move this after the out: below and also add a check for:

```
   if (NULL != file_vers) {
        loader_instance_heap_free(inst, file_vers);
   }
```

Reply:
Regarding 2607, yes you're correct. But 2506?

Reply:
Sorry, 2596 not 2506.